WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -197,47 +182,46 @@ private extension SupportTableViewController {
         }
 
         // Create and return table
-        let sections = [helpSection, forumsSection, informationSection, logOutSections].compactMap { $0 }
+        let sections = [forumsSection, informationSection, logOutSections].compactMap { $0 }
         return ImmuTable(sections: sections)
     }
 
-    // TODO - remove after FeatureFlag.wordPressSupportForum is removed
-    func oldTableViewModel() -> ImmuTable {
+    func jetpackAppTableViewModel() -> ImmuTable {
 
         // Help Section
         var helpSectionRows = [ImmuTableRow]()
-        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button", featureFlagSupportForum: featureFlagStore.value(for: FeatureFlag.wordPressSupportForum)))
+        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button", featureFlagSupportForum: isForumShown))
 
         if ZendeskUtils.zendeskEnabled {
-            helpSectionRows.append(HelpRow(title: LocalizedText.contactUs, action: contactUsSelected(), accessibilityIdentifier: "contact-support-button", featureFlagSupportForum: featureFlagStore.value(for: FeatureFlag.wordPressSupportForum)))
-            helpSectionRows.append(HelpRow(title: LocalizedText.myTickets, action: myTicketsSelected(), showIndicator: ZendeskUtils.showSupportNotificationIndicator, accessibilityIdentifier: "my-tickets-button", featureFlagSupportForum: featureFlagStore.value(for: FeatureFlag.wordPressSupportForum)))
+            helpSectionRows.append(HelpRow(title: LocalizedText.contactUs, action: contactUsSelected(), accessibilityIdentifier: "contact-support-button", featureFlagSupportForum: isForumShown))
+            helpSectionRows.append(HelpRow(title: LocalizedText.tickets, action: myTicketsSelected(), showIndicator: ZendeskUtils.showSupportNotificationIndicator, accessibilityIdentifier: "my-tickets-button", featureFlagSupportForum: isForumShown))
             helpSectionRows.append(SupportEmailRow(title: LocalizedText.contactEmail,
                                                    value: ZendeskUtils.userSupportEmail() ?? LocalizedText.emailNotSet,
                                                    accessibilityHint: LocalizedText.contactEmailAccessibilityHint,
                                                    action: supportEmailSelected(),
                                                    accessibilityIdentifier: "set-contact-email-button",
-                                                   featureFlagSupportForumEnabled: featureFlagStore.value(for: FeatureFlag.wordPressSupportForum)))
+                                                   featureFlagSupportForumEnabled: isForumShown))
```

Comment:
@guarani parameter `featureFlagSupportForum` for class `SupportEmailRow` similar to my comment on `HelpRow`